pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -23,6 +23,7 @@
 , typst-preview
 , autoPatchelfHook
 , zlib
+, libxcrypt-legacy
```

Comment:
Recently has been documented the review criteria for `vscode-extensions`, you might find it useful:

> https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vscode/extensions/README.md#conventions-for-adding-new-extensions

Consider:
> Move extension to a discrete directory whenever the extension needs extra parameters/packages (at top of the file) or other files (such as patches, update script, components). Global index file parameters/packages should be utilities shared by many extensions. Extension specific parameters/packages should not be in the global index page.